src/chains/EVMChain.ts

```diff
@@ -1,5 +1,4 @@
-import { BigNumber } from 'ethers'
-import { _Chain } from './Chain'
+import type { _Chain } from './Chain.js'
```

Comment:
why from Chain.js?

Reply:
Oh... that's a hot topic and will require a long answer 😂 
TL;DR This is needed for the new package/ts ESM configuration and setup.

Something to read https://github.com/microsoft/TypeScript/issues/49083